cypress/e2e/deletePost.spec.js

```diff
@@ -0,0 +1,11 @@
+describe('DELETE /posts/:id', () => {
+    it('перевірка стату коду', () => {

```

Comment:
```suggestion
    it('видалення посту за айдішкою', () => {
```
Я б вказував що саме мета тесту в описі